public/css/shorten_result.css

```diff
@@ -12,3 +12,14 @@
 .content-div {
     text-align: center;
 }
+
+.qrCodeContainer {
+    display: none;
+    width: 300px;
+    height: 300px;
```

Comment:
For consistency, `em` is preferred over `px`

Reply:
The reason I used px here is because the size of the QR code is defined in px. Would you still like me to change it?